app/src/androidTest/java/com/github/palFinderTeam/palfinder/profile/UIMockProfileServiceModule.kt

```diff
@@ -155,7 +157,7 @@ object UIMockProfileServiceModule {
         override suspend fun unfollowUser(user: ProfileUser, targetId: String): Response<Unit> {
             return try {
                 if (!user.canUnFollow(targetId)) {
-                    return Response.Failure("Cannot unfollow this user.")
+                    return Response.Failure("Cannot mute this meetup.")
```

Comment:
shouldn't we exchange the failure messages between unfollowUser and followUser ?